x-pack/platform/plugins/shared/task_manager/server/lib/create_managed_configuration.ts

```diff
@@ -201,10 +201,14 @@ function countErrors(
           isEsCannotExecuteScriptError(e) ||
           getMsearchStatusCode(e) === 429 ||
           getMsearchStatusCode(e) === 500 ||
+          getMsearchStatusCode(e) === 502 ||
```

Comment:
I thought we were going to change to check any status code (not just msearch/bulkUpdate) for these.  I was also thinking we wanted all >= 500, so would still want to check for 502/504.

Reply:
Oh, we can't use a generic check.
We get errors from 3 different sources and they report the status codes in different formats.
errors from savedObjectsClient(1),  msearch(2) and bulkUpdate(3).

bulkUpdate one is also important to detect the cluster block exception.

But I can change the check to 
`getMsearchStatusCode(e) === 429 || getMsearchStatusCode(e) >= 500`
`getBulkUpdateStatusCode(e) === 429 || getBulkUpdateStatusCode(e) >= 500`

Reply:
Ah, ok.  Your suggested change seems good then.  Since polling/claiming requires those two calls, presumably that's enough to set things off ...

Reply:
done